Fixed slashes in file tests when developing on Windows machines

odysseus654 opened the pull request:
The tests have been assuming the results would contain forward slashes, however on Windows machines they were universally failing with backslashes. This uses String.join(File.separator) on the failing tests to (hopefully) make the code more platform-independent.

## Status
READY

## Does this PR contain migrations?
NO

## Before You Submit Your PR:
- [X] Have you announced your PR on the comixed-dev mailing list?
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactors existing code (code changes for efficiency or maintainability)
- [ ] Security fix (be sure to include the CVE in the commit message) 
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

mcpierce reviewed (changes requested):
Looks good. Please make the following change to the commit message:

**Changed adaptor tests to use platform-specific separators [<span>#</span>1845]**

and be sure to add yourself to the CONTRIBUTORS.md file to get credit for your contributions. And thank you!

mcpierce reviewed (changes requested):
Please update the git commit message to be:

**Changed adaptor tests to use platform-specific separators [\#1845]**

so that it conforms to the project commit standard.

mcpierce commented:
@odysseus654 The only thing that needs changing on this PR is to update the commit message and to rebase it. Do you want to update the PR or close it out?

odysseus654 commented:
Okay, finally getting back to this.  And didn't realize you opened a bug report for this.  Sounds good.

odysseus654 committed:
Fixed slashes in file tests when developing on Windows machines [#1845]

The tests have been assuming the results would contain forward slashes,
however on Windows machines they were universally failing with backslashes.
This uses String.join(File.separator) on the failing tests to (hopefully)
make the code more platform-independent.

odysseus654 commented:
I added the CONTRIBUTORS and left the additional description in the commit message (especially as it wasn't in the bug report), this acceptable?
